lib/stylePlugins/scoped.ts

```diff
@@ -35,6 +36,13 @@ export default postcss.plugin('add-id', (options: any) => (root: Root) => {
             n.spaces.before = n.spaces.after = ''
             return false
           }
+
+          // in newer versions of sass, /deep/ support is also dropped, so add a ::-v-deep alias
+          if (n.type === 'pseudo' && n.value === '::-v-deep') {
```

Comment:
why is it named `::-v-deep` instead of `::v-deep`?

Reply:
CSS prefixes typically start with `-`, e.g. `-webkit-`, so I think we can follow this convention to indicate that it's not a standard selector.

Reply:
I think it makes it more verbose than necessary. It's not a property name, and is also prefixed with `::` already.